noir-projects/aztec-nr/aztec/src/utils/to_bytes.nr

```diff
@@ -0,0 +1,25 @@
+pub fn arr_to_be_bytes_arr<let L: u32>(fields: [Field; L]) -> [u8; L * 32] {
```

Comment:
Moved these functions from `log_traits.nr` + refactored them such that they work with arrays instead of slices (we no longer need slices here with arithmetic over generics).